assets/components/episode-search.js

```diff
@@ -0,0 +1,218 @@
+import jQuery from 'jquery';
```

Comment:
It would be nice to get some unit testing on this class.

I can add that to my list of things to do if you're not dying to get this PR merged unless you want to have a go at it.

Reply:
I agree, this could be a good starting point for testing web components ([reference](https://developer.salesforce.com/blogs/2019/03/unit-test-lightning-web-components-with-jest.html)). I'm going to merge it for now but I'll try to get a PR starting tomorrow with some tests.

Reply:
History suggests that such "temporary fixes" and "I'll do it later" results in no additional work.

Something about the road to Hell being paved with Good Intentions.

Reply:
You are correct, I hate work.